apps/service/src/app/(blank)/long/_components/VideoPlaybackManager.tsx

```
@@ -79,6 +70,17 @@ export function VideoPlaybackManager({ thumbnail }: VideoPlaybackManagerProps) {
+    return (
+      <div
+        className="flex w-full items-center justify-center bg-black text-white"
+        style={{ aspectRatio: '16/9' }}
+      >
+        유효하지 않은 영상입니다.
+      </div>
+    );
+  }
+
   // 재생 목록(찜 목록 등) 자동 재생 훅
   const handleVideoEnd = usePlaylistAutoPlay(videoId);
```

Biome (2.4.6)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)